### PR TITLE
Added feature to make sure users have to agree to the General terms a…

### PR DESCRIPTION
**Pull request by KasperVaessen**
Users now have to agree to the general terms and conditions before they can check out. This is checked on the back end. It looks like this: 
image
image

The link directs to wisv.ch/GTC. Here the GTC will be placed once it is updated according to the comments made at the GA.

### CODE REVIEW

**Comment by julian9499** [comment]
This can be merged once the general terms and conditions are available. I recommend linking this properly with the link of the document on the site itself instead of using mand.

**Comment by JoepdeJong** [comment]
@julian9499 did you test the PR?

**Comment by KasperVaessen** [comment]
The document has been uploaded on the website. I also edited the link in the html.

File: src/main/resources/static/css/wisvch-dashboard.css [hunk 1]
```diff
@@ -153,6 +153,12 @@ textarea:focus {
     transition: box-shadow 0.15s linear, -webkit-box-shadow 0.15s linear
 }
 
+#checkoutShoppingBasket {
+    margin-left: auto;
+    margin-right: 0;
+    float: right;
+}
+
```
Comment (julian9499) [inline src/main/resources/static/css/wisvch-dashboard.css:161]: Why is this here?

File: src/main/java/ch/wisv/events/core/model/order/OrderProductDto.java [hunk 2]
```diff
@@ -10,11 +10,13 @@ public class OrderProductDto {
      * Field product should contain Product keys and number of webshop of that product.
      */
     HashMap<String, Long> products;
+    Boolean agreedGTC;
```
Comment (julian9499) [inline src/main/java/ch/wisv/events/core/model/order/OrderProductDto.java:13]: Is it necessary to use `Boolean` here?

**Comment by julian9499** [comment]
I didn't yet, @JoepdeJong. @KasperVaessen if you answer the two questions that I have I will test it

File: src/main/resources/static/css/wisvch-dashboard.css (hunk 1 above)
Comment (KasperVaessen) [reply src/main/resources/static/css/wisvch-dashboard.css:161]: to align the button on the right, since the addition of the checkbox moved it more left.
image, however i noticed that only the float is neccesary, so I will remove the margins.

File: src/main/java/ch/wisv/events/core/model/order/OrderProductDto.java (hunk 2 above)
Comment (KasperVaessen) [reply src/main/java/ch/wisv/events/core/model/order/OrderProductDto.java:13]: Instead of using bool you mean? I always learned to only use Object types instead of native types in models in spring, because native types often cause trouble with the database.
